### PR TITLE
Add homework files

### PR DESCRIPTION
**Pull request by gulyener**

### CODE REVIEW

File: Week1/homework/app.js [hunk 1]
```diff
@@ -1,11 +1,303 @@
...
+};
+
+window.addEventListener('load', getBookInfo);
+window.addEventListener('load', getBookCovers);
 
-  // Replace with your own code
-  console.log(bookTitles);
-}
+// I used element.attribute instead of setAttribute() throughout this homework
+// it feels easier to do it that way, but is it bad practice?
```
Comment (nudge) [inline Week1/homework/app.js]: Using `element.attribute` is perfectly fine in practice!

File: Week1/homework/app.js [hunk 2]
```diff
@@ -1,11 +1,303 @@
...
+  'Crime and Punishment': './images/crime_and_punishment.jpeg',
+  'Moby Dick': './images/moby_dick.jpg',
+  Hamlet: './images/hamlet.jpg',
+  'Madame Bovary': './images/madame_bovary.jpg',
+  'Jane Eyre': './images/jane_eyre.jpg',
+  'Pride and Prejudice': './images/pride_and_prejudice.jpg',
+  'Anna Karenina': './images/anna_karenina.jpeg',
+};
+
+// 1.8 I could not do this with Object.keys(objectName) I hope I can learn it from someone
```
Comment (nudge) [inline Week1/homework/app.js]: A way to do this is to set the `id` attribute on the `li` element you create in `getBookInfo()`, then in `getBookCovers()` you can iterate over the `li` elements, grab the `id`, and match it with `myBooks`.

File: Week1/homework/.vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  "configurations": [
+    {
+      "type": "chrome",
+      "request": "launch",
+      "name": "Launch Chrome against localhost",
+      "url": "http://localhost:8080",
+      "webRoot": "${workspaceFolder}"
+    }
+  ]
+}
```
Comment (nudge) [inline Week1/homework/.vscode/launch.json]: You don't need to check this file in, you can remove it from the pull request

File: Week1/homework/app.js [hunk 4]
```diff
@@ -1,11 +1,303 @@
...
+//     li.innerHTML = bookTitles[i];
+//     ul.appendChild(li);
+//   }
+// };
+// document.getElementById('renderList').appendChild(ul);
+
+// 1.4
+
+const myBooks = {
+  'War and Peace': {
```
Comment (nudge) [inline Week1/homework/app.js]: Usually keys are used as identifiers. I've noticed you're using them here for display as titles of the books. A more flexible approach is to keep the key as an identifier, e.g. `war_and_peace`, then add a key within the object like `title: 'War and Peace'`.

File: Week1/homework/app.js (hunk 1 above)
Comment (gulyener) [reply Week1/homework/app.js]: Okay that's good to know thank you for the information

**Pull request closed without merging**